fdk.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.summary     = "Ruby FDK for Fn Project"
   s.description = "Ruby Function Developer Kit for Fn Project."
   s.license     = "Apache-2.0"
-  s.required_ruby_version = ">= 2.4"
+  s.required_ruby_version = "= 2.7.3"
```

Comment:
Setting this required version precludes running this on an Oracle base image.
Even if we choose to maintain a ruby 2.7 distribution ourselves I think that it makes sense to keep the minimum requirement for the FDK, as a library, to something that can run on OL, perhaps `">= 2.5"`

Reply:
I stand corrected... OL8 has a Ruby 2.7 stream

Reply:
I added this change as Circle CI build was failing because of some dependency issue with 2.4.
When I changed to latest supported one it worked.